datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/OwnerUtils.java

```diff
@@ -195,7 +196,16 @@ private static void removeOwnersIfExists(
   }
 
   public static void validateAuthorizedToUpdateOwners(
-      @Nonnull QueryContext context, Urn resourceUrn) {
+      @Nonnull QueryContext context, Urn resourceUrn, EntityClient entityClient) {
+
+    if (resourceUrn.getEntityType().equals(Constants.GLOSSARY_TERM_ENTITY_NAME)
```

Comment:
Can we consolidate this logic which seem duplicated across multiple util classes. Some common util class is likely better, such as AuthorizationUtils.java

Reply:
@david-leifker - I have made this change. Please check.